libafl/Cargo.toml

```diff
@@ -31,6 +31,7 @@ errors_backtrace = ["backtrace"] # Create backtraces at Error creation
 cmin = ["z3"] # for corpus minimisation
 corpus_btreemap = [] # Switches from HashMap to BTreeMap for CorpusId
 gzip = ["miniz_oxide"] # Enables gzip compression in certain parts of the lib
+timestamp = ["chrono"]
```

Comment:
Would name it logger_timestamp or something? Also it should depend on "std" and should have a comment that explains it